namelint.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# Copyright 2008 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+import argparse
+
+from lib.linters import build_linters, severity as sev
+from lib.naming import Naming
+
+__author__ = 'mikeelkin2@fb.com (Mike Elkin)'
+
+
+def main():
```

Comment:
We don't use gflags for python which is why the code came like this. I'm prefer to have the gflags cutover be separate. We did notice that gflags feels very different to argparse, which is more commonly adopted in the community.